src/cablab/providers/albedo.py

```diff
@@ -33,15 +29,15 @@ def get_variable_descriptors(self):
         return {
             VAR_NAME_BRIGHT: {
                 'data_type': numpy.float32,
-                'fill_value': numpy.nan,
+                'fill_value': FILL_VALUE,
                 'units': '-',
```

Comment:
I also have a question regarding the FILL Values. Do you always keep the FillValue of the original data or should we prefer to have a common CABLAB Float Fillvalue?